Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -107,6 +111,11 @@ Tileset::Tileset(
   if (ionAssetID > 0) {
     CESIUM_TRACE_USE_TRACK_SET(this->_loadingSlots);
     this->notifyTileStartLoading(nullptr);
+    if (this->_externals.pTileOcclusionProxyPool) {
+      // TODO: find a reasonable number
+      this->_externals.pTileOcclusionProxyPool->initPool(
+          options.occlusionPoolSize);
```

Comment:
Perhaps the renderer should have control over the pool size?

Reply:
The pool size here comes from the `TilesetOptions` which is configurable on the renderer, would it be better if it was determined somewhere else? In Cesium for Unreal, this is configurable in the `Cesium3DTileset` details panel.

Reply:
It seems like it might be cleaner if the renderer were solely responsible for creating, sizing, and destroying the pool. Get rid of `initPool` completely.

Reply:
That makes sense, but that would also shift the responsibility of managing the pool (allocating from and freeing to the pool) to the renderer right? Currently the renderer only has to provide an implementation for creating individual proxies, but it doesn't have to manage the mapping to specific tiles and manage a free list.